purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallData.kt

```diff
@@ -44,6 +70,10 @@ data class PaywallData(
 
     @SerialName("localized_strings_by_tier") internal val localizationByTier:
     Map<String, Map<String, LocalizedConfiguration>> = emptyMap(),
+
+    @SerialName("zero_decimal_place_countries")
+    @Serializable(with = GoogleListSerializer::class)
+    val zeroDecimalPlaceCountries: List<String> = emptyList(),
```

Comment:
Also need to add this to API tests

Reply:
I don't see any explicit API testers for PaywallData, but the sample paywalls in SamplePaywalls.kt serves this purpose. Is there somewhere else this should be added?